Content/posts/update-03-october-2020.md

```diff
@@ -0,0 +1,171 @@
+
+After these changes were merged [`carton` 0.7 was released](https://github.com/swiftwasm/carton/releases/tag/0.7.0)
+and is now available via Homebrew on macOS.
+
+After the release, [@carson-katri](https://github.com/carson-katri) merged a PR that greatly improves
+command-line experience with `carton test`. It builds on his previous work on `carton dev` with
+compiler diagnostics parsing and error highlighting. With this new change XCTest output
+is also parsed, reformatted, and highlighted in a nice summary view.
+
+<br />
+
+![Pretty-printed test summary in `carton`](./images/update-03-carton-test-summary.png width=100%)
+
+<br />
+<br />
+
+![Pretty-printed test errors in `carton`](./images/update-03-carton-test-errors.png width=100%)
+
+<br />
+<br />
+
+### GitHub Actions
+
+Thanks to the fact that `carton` is now available in a Docker image, it can now be used in
+our [`swiftwasm-action`](https://github.com/swiftwasm/swiftwasm-action), which previously only
+contained the plain SwiftWasm toolchain without any additional tools. This action now
+invokes `carton test` by default on a given repository during a GitHub Actions run, but you can
+customize it and call any other command. For example, you could run `carton bundle` with `swiftwasm-action`,
+and then use the [`peaceiris/actions-gh-pages`](https://github.com/peaceiris/actions-gh-pages) step
+to deploy the resulting bundle to [GitHub Pages](https://pages.github.com/).
+
+## Libraries
+
+## JavaScriptKit
+
+[JavaScriptKit 0.8.0](https://github.com/swiftwasm/JavaScriptKit/releases/tag/0.8.0) has been
+released. As mentioned in our previous issue, it introduces a few enhancements and deprecations,
+and is a recommended upgrade for all users.
+
+### Tokamak
+
+[@Cosmo](https://github.com/Cosmo) pointed out that SwiftUI apps should change almost the whole
+palette when switching between color schemes. As he discovered when working on his
+[OpenSwiftUI](https://github.com/Cosmo/OpenSwiftUI/) and [SwiftUIEmbedded](https://github.com/Cosmo/SwiftUIEmbedded)
+projects, the implementation of the `Color` type is much more subtle than we originally anticipated.
+This was resolved by [@carson-katri](https://github.com/carson-katri) in
+[#291](https://github.com/TokamakUI/Tokamak/issues/291) and is now available in the `main` branch.
+The plan is to merge a few more changes and bugfixes, and to tag a new release of Tokamak soon
+after that.
+
+## Upstream PRs
+
+### Swift Driver
+
+[@owenv](https://github.com/owenv) merged [a PR to the Swift Driver
+project](https://github.com/apple/swift-driver/pull/315) implementing WebAssembly toolchain
+support. This mirrors our existing [C++ implementation in the legacy
+driver](https://github.com/swiftwasm/swift/blob/swiftwasm/lib/Driver/WebAssemblyToolChains.cpp), and
+it's great that the new parts of the Swift compiler rewritten in Swift are going to support
+WebAssembly too. While Swift Driver in that enabled in any toolchain by default, we're definitely
```

Comment:
I'm not quite sure what the beginning of this sentence is saying. Maybe:
```suggestion
WebAssembly too. While Swift Driver isn't enabled in any toolchain by default, we're definitely
```

Reply:
Thanks, that's updated in d02849836d88bd744f89b21420920409ae563151.